libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -701,14 +707,18 @@ internal class MapboxTripSession(
     }
 
     /**
-     * Invalidate latest banner instruction. To get the latest banner instruction wrapper call
-     * [BannerInstructionEvent.latestInstructionWrapper]
+     * Invalidate latest banner and voice instruction. To get the latest banner instruction wrapper call
+     * [BannerInstructionEvent.latestInstructionWrapper], to get the latest voice instruction
+     * call [lastVoiceInstruction]
      */
     private fun invalidateLatestInstructions(
-        latestInstructionWrapper: BannerInstructionEvent.LatestInstructionWrapper?
+        latestInstructionWrapper: BannerInstructionEvent.LatestInstructionWrapper?,
+        voiceInstruction: VoiceInstructions?,
```

Comment:
Is it right for the index to be wrapped in banner instruction event if it's responsible both for banner and voice instruction?
I'd keep them all together.

Reply:
The index is taken from `BannerInstruction#getIndex` and later the SDK re-uses that `index` value for voice instruction as well. Not sure why it's done so. (the index of voice and banner instruction must be the same, but why it's taken from `BannerInstruction#getIndex` is not clear)
@LukasPaczos do you have the context?
We can address the issue separately

Reply:
cut `NAVAND-964`